spinnaker/src/main/java/org/bytedeco/spinnaker/presets/Spinnaker_C.java

```diff
@@ -93,6 +93,9 @@ public void map(InfoMap infoMap) {
                 .put(new Info("SPINC_CALLTYPE").cppTypes().annotations().cppText(""))
                 .put(new Info("SPINC_NO_DECLSPEC_STATEMENTS", "EXTERN_C").define())
                 .put(new Info("SPINNAKERC_API").cppTypes("_spinError").cppText("enum _spinError").define())
+                // spinEnum is not getting generated so force it using enum name "_spinError"
+                //    and give correct generated name "spinError"
+                .put(new Info("_spinError").enumerate().valueTypes("spinError"))
```

Comment:
That's probably because of the line above. Try to remove that line, and move "SPINNAKERC_API" to the right of "SPINC_CALLTYPE" instead.

Reply:
No sure what you mean to do with "SPINNAKERC_API"?

The line in preset
```
new Info("SPINNAKERC_API").cppTypes("_spinError").cppText("enum _spinError").define()
```
is due to line in C code:
```C
#define SPINNAKERC_API SPINC_IMPORT_EXPORT spinError SPINC_CALLTYPE
```
the usge in C code is:
```C
SPINNAKERC_API spinErrorGetLast(spinError* pError);
```
That is `SPINNAKERC_API` is uised as return value

Reply:
That's fine, it's going to expand it to "spinError", as long as you make sure it doesn't try to add types to all other macros as well (SPINC_IMPORT_EXPORT, SPINC_CALLTYPE, and whatever else there is).

Reply:
I am not clear if you want me to change something? Is this PR OK?
I can use this Spinnaker preset in a demo code that interacts with a camera, so in that sense it is fine.

Reply:
Please try to replace these 3 lines:
```java
.put(new Info("SPINC_CALLTYPE").cppTypes().annotations().cppText(""))
.put(new Info("SPINNAKERC_API").cppTypes("_spinError").cppText("enum _spinError").define())
.put(new Info("_spinError").enumerate().valueTypes("spinError"))
```
with this single line:
```java
.put(new Info("SPINC_CALLTYPE", "SPINNAKERC_API", "SPINC_IMPORT_EXPORT").cppTypes().annotations().cppText(""))
```

Reply:
OK. I have to run. I will try tomorrow